### PR TITLE
Fix payment status

### PR DESCRIPTION
**Pull request by harshvitra**
Fixed payment status

### CODE REVIEW

File: src/features/donations/components/PaymentFunctions.ts [hunk 1]
```diff
@@ -271,13 +271,15 @@ export async function payDonationFunction({
...
-      } else if (paidDonation.paymentStatus === 'success' || paidDonation.paymentStatus === 'pending' ||
-      paidDonation.data.status === "success" || 
-      paidDonation.data.status === "paid" ||
-      paidDonation.data.paymentStatus === "paid") {
+        return paidDonation;
+      } else if (paidDonation.paymentStatus === 'success'
+        || paidDonation.paymentStatus === 'pending'
+        || paidDonation.paymentStatus === 'paid'
+        || paidDonation.status === 'success'
+        || paidDonation.status === 'pending'
```
Comment (norbertschuler) [inline src/features/donations/components/PaymentFunctions.ts:279]: @harshvitra I added this just to be sure as `paidDonation.paymentStatus === 'pending'` was also requested before -but I am not sure about the status model currently used for payments.

**Review by norbertschuler — APPROVED** [review]
This works for me, but currently on the development backend, successful donations are still marked as "not finalized", so we probably are not yet finished - maybe this can be solved in the backend?

Bildschirmfoto 2021-06-06 um 15 56 46

File: src/features/donations/components/PaymentFunctions.ts (hunk 1 above)
Comment (norbertschuler) [reply src/features/donations/components/PaymentFunctions.ts:279]: Credit card payment on staging (not development) backend are getting finalized now, Paypal not.

Comment (sagararyal) [reply src/features/donations/components/PaymentFunctions.ts:279]: This is prob because of missing web hooks to staging from PayPal.

**Pull request merged**